src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Templates/Templates/DefaultTemplates/Startup.cs

```diff
@@ -28,7 +29,16 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllersWithViews(ConfigureMvcOptions);
+            services.AddControllersWithViews(ConfigureMvcOptions)
+                // Newtonsoft.Json is added for compatibility reasons
+                // The recommended approach is to use System.Text.Json for serialization
+                // Visit the following link for more guidance about moving away from Newtonsoft.Json to System.Text.Json
+                // https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to?pivots=dotnet-5-0
```

Comment:
You can remove the /en-us/ part of docs.microsoft.com URLs to make them automatically redirect to the locale that's right for the person following the line. Also, the query string here is probably not necessary and removing it will make the URL shorter.